packages/@aws-cdk/aws-servicecatalog/package.json

```diff
@@ -71,30 +72,52 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
-    "@types/jest": "^26.0.23",
+    "@aws-cdk/assert-internal": "0.0.0",
+    "@types/jest": "^26.0.22",
+    "@typescript-eslint/eslint-plugin": "^4.24.0",
+    "@typescript-eslint/parser": "^4.24.0",
     "cdk-build-tools": "0.0.0",
+    "cdk-integ-tools": "0.0.0",
     "cfn2ts": "0.0.0",
-    "pkglint": "0.0.0",
-    "@aws-cdk/assert-internal": "0.0.0"
+    "eslint": "^7.26.0",
+    "eslint-config-standard": "^16.0.2",
+    "eslint-plugin-import": "^2.23.2",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.3.1",
+    "pkglint": "0.0.0"
```

Comment:
What's with all these `eslint` changes? These are centralized in the CDK build tools, they should _not_ be here.

Reply:
I think these were leftover from our original branch when we had just a shell and imported them to make VS Code happy since we were breaking all the linting rules, removed